platform/core.network/test/unit/src/org/netbeans/core/network/utils/FakeDns.java

```diff
@@ -172,6 +176,22 @@ public void delayAnsweringBy(long msWait ) {
         return null;
     }
 
+    @Override
+    Callable<InetAddress[]> createDnsTimeoutTask(String host) {
```

Comment:
@lbruun, please take a look. Hopefully I changed no semantics, but now the test passes on JDK11. Instead of injecting something into `sun.*.net.*` packages, I inject different implementation of `createDnsTimeoutTask` in the test.